pages/_app.js

```diff
@@ -1,7 +1,13 @@
-import '../styles/globals.css'
+import { GlobalStyle } from '../components/GlobalStyle';
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return <Component {...pageProps} />;
+  return (
```

Comment:
You seem to have a double return here. The code will stop after the first return and not continue to the second one.